news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "osu! originals - osu!catch World Cup 2023"
+date: 2023-06-28 11:00:00 +0000
+---
+
+If you missed South Korea's intense comeback from the loser's bracket of the [osu!catch World Cup](/wiki/Tournaments/CWC) last weekend, you might've also missed **5 brand-new songs** created specifically for the tournament!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+Those [**osu! originals**](/wiki/osu!_originals) were painstakingly produced by both new and veteran artists for [2023's osu!catch World Cup](/wiki/Tournaments/CWC/2023), so check out the songs for yourself (...and maybe map one, if you're into that sort of thing):
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="LINK" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### [LV.4 - Chemical Diving](https://osu.ppy.sh/beatmapsets/2007952)
+
+Back for their third world cup in a row, [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214) is practically a permanent resident of our official tournaments (which is a good thing, obviously).
+
+[Chemical Diving](https://osu.ppy.sh/beatmapsets/2007952) by [Rocma](https://osu.ppy.sh/users/566276) and [Du5t](https://osu.ppy.sh/users/6053071) slotted in as a *Hard Rock* pick for the [Finals mappool](/wiki/Tournaments/CWC/2023#finals), and proved itself as a testing ground for this year's strongest teams. In the decisive round showcased above, the US barely overtook South Korea by a slim 17,000 points!
+
+acted as a *Free Mod* pick in the [Grand Finals mappool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
```

Comment:
```Suggestion
*Chemical Diving* acted as a *Free Mod* pick in the [Grand Finals mappool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
```

Reply:
this paragraph wasnt supposed to be here lol